README.md

```diff
@@ -1 +1,64 @@
-# remember-pi-project
+
+# REMEMBER
+
```

Comment:
Logo abaixo do título do projeto dá pra colocar badges com os nomes das tecnologias, as mesmas que aparecem na seção de tecnologias, a diferença é que aparecendo logo no início dá uma visão do que esperar encontrar no projeto, dá uma olhada nesse readme aqui:
https://github.com/alexa2me/express-cookenu-API/blob/main/README.md